pubspec.yaml

```diff
@@ -13,8 +13,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  flutter_web_plugins:
+    sdk: flutter
   freezed_annotation: ^2.0.1
   json_annotation: ^4.8.0
+  js: ^0.6.7
```

Comment:
This is deprecated now... But will keep using it for now and move to the `web` plugin in the future.

Reply:
You're correct! noticed that after my initial comment. Need to revisit this before shipping this in any stable form. Thanks for checking!!